src/js/fetchData.js

```diff
@@ -1,13 +1,13 @@
 const THE_PROJECT_MYANMAR_SERVICE =
-  'https://projectmyanmar.herokuapp.com/projectmm/sections';
+  'https://projectmyanmar.herokuapp.com/projectmm';
 const API_SECRET = '442A472D4B615064';
```

Comment:
Can you replace the url & secret w/ 
- process.env.REACT_APP_BACKEND_API_BASE_URL
- process.env.REACT_APP_BACKEND_API_SECRET

Reply:
will do